.changeset/lucky-cups-play.md

```diff
@@ -0,0 +1,16 @@
+---
+"create-fuels": patch
+"@fuel-ts/abi-coder": patch
+"@fuel-ts/contract": patch
+"@fuel-ts/account": patch
+"@fuel-ts/address": patch
+"@fuel-ts/program": patch
+"@fuel-ts/errors": patch
+"@fuel-ts/hasher": patch
+"@fuel-ts/merkle": patch
+"@fuel-ts/script": patch
+"@fuel-ts/utils": patch
+"@fuel-ts/math": patch
+---
+
+chore: add initial `depcheck` using knip
```

Comment:
Strange one [echoing it out](https://github.com/FuelLabs/fuels-ts/pull/2026/files#diff-0457276889ca7e807204647f38c66df82b275dc0b87b310ed2675f4f2e344e62R77) made it [pass](https://github.com/FuelLabs/fuels-ts/actions/runs/8634484502/job/23670036938?pr=2026)   🤪 

I'm ok with keeping that change should this issue re-occur...

Reply:
Never seen this one before. 🤔

Reply:
My only suspicion is that it could have been caching the old changeset as this changeset is actually a completely new file i.e. I re-did `pnpm changeset add` otherwise I am not sure what could've caused it.